### PR TITLE
add Mangata X

### PR DESCRIPTION
**Pull request by Tomen**

### CODE REVIEW

**Review by PierreBesson — APPROVED** [review]

**Comment by PierreBesson** [comment]
Everything look good to me. @Tomen please sign the CLA and we can merge.

File: chain_info/Mangata_X.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  ed: 10000000000
+  decimals: 18
+  token: MGX
+  ss58_format: 42
+contacts:
+  - type: email
+    contact: "hello@mangata.finance"
+rpc_endpoints:
+  - name: Parity
+    url: wss://kusama-rpc.mangata.online:443
```
Comment (chevdor) [inline chain_info/Mangata_X.yaml:27]: This is definitely NOT a Parity endpoint.

**Review by chevdor — CHANGES_REQUESTED** [review]
Please adjuste the name of the endpoint

File: chain_info/Mangata_X.yaml (hunk 1 above)
Comment (Tomen) [reply chain_info/Mangata_X.yaml]: I changed line 26

https://github.com/paritytech/polkadot_network_directory/pull/60/commits/4d671b99cb19ccf0a171393e7981f322a9d04d9e

**Comment by Tomen** [comment]
added a commit that fulfilled the change request

**Review by chevdor — APPROVED** [review]

**Pull request closed without merging**